redis.go

```diff
@@ -62,6 +62,10 @@ func (om *OpenMock) SetRedis() {
 	}
 }
 
+func (om *OpenMock) RedisDo(commandName string, args ...interface{}) (reply interface{}, err error) {
+	return om.redis.Do(commandName, args...)
```

Comment:
om.redis may be `nil`, this may need a nil check here.

Reply:
:+1: